autoload/sy.vim

```diff
@@ -96,7 +96,13 @@ function! sy#start() abort
 
   let b:sy.id_top = (g:id_top - 1)
 
-  silent! doautocmd <nomodeline> User Signify
+  if exists('#User#Signify')
+    if v:version > 703 || v:version == 703 && has('patch442')
```

Comment:
Might be worth caching this check when the script is loaded.  Other than that, looks good.